includes/class-wc-payments-account.php

```diff
@@ -709,37 +751,28 @@ public function maybe_redirect_after_plugin_activation() {
 			return false;
 		}
 
-		$account = $this->get_cached_account_data();
```

Comment:
hmm, why is it removed?

Reply:
Because it is tested by:
```
// If everything is in working order, don't redirect.
if ( $this->has_working_jetpack_connection() && $this->is_stripe_account_valid() ) {
```